root/artist/edit_form.tt

```diff
@@ -97,16 +101,93 @@
+             { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('This field is not a place to store general background information.
+              That kind of information should go in an {doc|annotation}.',
+             { doc => { href => doc_link('Annotation'), target => '_blank' } }) %]
+      </p>
+    </div>
+
     [%- type_bubble(form.field('type_id'), artist_types) -%]
 
+    <div class="bubble" id="gender-bubble">
+      <p>
+        [% l('The {doc|gender} field describes the gender with which a person or character identifies.',
+             { doc => { href => doc_link('Artist#Gender'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Gender'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="area-bubble">
+      <p>
+        [% l('The {doc|area} indicates the area the artist is primarily identified with.',
+             { doc => { href => doc_link('Artist#Area'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Main_area'), target => '_blank' } }) %]
+      </p>
+      [% area_bubble_selection() %]
+    </div>
+
     [%- ipi_bubble() -%]
     [%- isni_bubble() -%]
-  </div>
 
+    <div class="bubble" id="begin-end-date-bubble">
+      <p>
+        [% l('The {doc|begin and end dates} describe when the artist started and stopped existing.',
+             { doc => { href => doc_link('Artist#Begin_and_end_dates'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Begin_and_end_areas/dates'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="begin-end-area-bubble">
+      <p>
+        [% l('The begin and end areas describe where the artist was when it started and stopped existing.') %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Begin_and_end_areas/dates'), target => '_blank' } }) %]
+      </p>
+      [% area_bubble_selection() %]
+    </div>
+
+    <div class="bubble" id="external-link-bubble">
+      <p>
+        [% l('External links are URLs associated with the artist.
+              These can include the artist’s official homepage and social networking pages, Wikipedia entries,
+              streaming services, and third‐party pages like biographies, interviews, and fan sites.') %]
+      </p>
+      <p>
+        [% l('Please don’t add a Wikipedia page if a Wikidata item linking to the same article already exists.') %]
```

Comment:
I mentioned this here since it happens frequently and (as I understand it) results in the wrong article being shown to users who aren't using the same language as the article link.